src/list/manage/index.js

```diff
@@ -34,11 +36,22 @@ let store;
   store = createStore(reducer, preloadedState, applyMiddleware(
     thunk, telemetryLogger, saveSort, openLink,
   ));
-  store.dispatch(listItems());
+  await store.dispatch(listItems());
   initializeMessagePorts(store);
   store.dispatch(getProfile());
   applyQueryFilter();
 
+  // if no filter check ensure that we have a default item selected
+  const params = new URLSearchParams(window.location.search);
+  const filter = params.get("filter");
+  if (!filter) {
```

Comment:
I think users will expect the first item from a query-based filter to be selected.

I think it's pretty doable, though it's probably going to feel like dupicate code from [all-items](https://github.com/mozilla-lockbox/lockbox-addon/blob/c15e67d23c46152b8d549f80d17b457ebba82526/src/list/manage/containers/all-items.js#L16-L22). 

Note that `state.filter.userEntered` will be false for a query-based filter.